### PR TITLE
Minor doc fix for PR#100

### PR DESCRIPTION
**Pull request by XianXing**
1. Adding the format example for ```featureShardIdToInterceptMap```.
2. Reformat the code.

### CODE REVIEW

File: photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala [hunk 1]
```diff
@@ -100,7 +104,6 @@ object Params {
               s"are assumed to be flat directories of input files (e.g., inputDir/input-data-files). " +
               s"Default: ${defaultParams.dateRangeDaysAgoOpt}.")
           .action((x, c) => c.copy(dateRangeDaysAgoOpt = Some(x)))
-        .action((x, c) => c.copy(dateRangeOpt = Some(x)))
```
Comment (XianXing) [inline photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala]: Here is a bug introduced in PR #97.

File: photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala [hunk 2]
```diff
@@ -82,10 +86,10 @@ object Params {
...
     val parser = new OptionParser[Params]("GLMix-Scoring-Params") {
       opt[String]("input-data-dirs")
-        .required()
-        .text("Input directories of data to be scored. Multiple input directories are also accepted if they are " +
-          "separated by commas, e.g., inputDir1,inputDir2,inputDir3.")
-        .action((x, c) => c.copy(inputDirs = x.split(",")))
+          .required()
+          .text("Input directories of data to be scored. Multiple input directories are also accepted if they are " +
+              "separated by commas, e.g., inputDir1,inputDir2,inputDir3.")
+          .action((x, c) => c.copy(inputDirs = x.split(",")))
```
Comment (joshvfleming) [inline photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala]: @XianXing I can't find anything about this in the style guide, but I believe the previous indentation style (2-space) is the generally accepted way when you have a multi-line method chain like this.

Comment (XianXing) [inline photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala]: Thanks for pointing it out. 

The continuation indent could be confusing sometimes, for example, for method/class's parameters the continuation indent is recommended to be 4-space (https://github.com/databricks/scala-style-guide#indent), while it looks like for most other cases it should be 2-space, as you suggested. But unfortunately Intellij doesn't allow us to set the continuation indent case by case, so we need to pay extra attention to it each time.

**Comment by joshvfleming** [comment]
LGTM

**Pull request merged**